### PR TITLE
feat: add section about remote images to assets

### PR DESCRIPTION
**Pull request by Princesseuh**
#### What kind of changes does this PR include?

- New or updated content

#### Description

https://github.com/withastro/astro/pull/7778 adds support for remote images, this PR updates the mention of remote images to specify that they'll be processed and document the two new options that decide which images are allowed to be processed

Sarah's 3.0 assets guide rewrite this part completely, so this is only temporary content until 3.0 releases

### CODE REVIEW

File: src/content/docs/en/guides/assets.mdx [hunk 1]
```diff
@@ -204,6 +204,34 @@ const allBlogPosts = await getCollection("blog");
...
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  image: {
+    remotePatterns: [{ hostname: "*.akamaihd.net" }],
+  }
+});
+```
+
+The protocol, hostname, pathname, and port of the image URL can be checked using this method. For instance, you could allow only images being served from HTTPs using `{ protocol: "https" }`
```
Comment (sarah11918) [inline src/content/docs/en/guides/assets.mdx:233]:
```suggestion
The protocol, hostname, pathname, and port of the image URL can be checked using this method. For instance, you could allow only images being served from HTTPs using `{ protocol: "https" }`.

See [`image.domains`](/en/reference/configuration-reference/#imagedomains) and [`image.remotePatterns`](/en/reference/configuration-reference/#imageremotepatterns) for detailed configuration instructions.
```
Note, these links will work once the accompanying config-reference docs are merged!

**Review by sarah11918 — APPROVED** [review]
Approving this for merge-on-release. After the ci docs, the config reference links should be fine.

File: src/content/docs/en/guides/assets.mdx [hunk 2]
```diff
@@ -204,6 +204,36 @@ const allBlogPosts = await getCollection("blog");
...
+export default defineConfig({
+  image: {
+    remotePatterns: [{ hostname: "*.akamaihd.net" }],
+  }
+});
+```
+
+The protocol, hostname, pathname, and port of the image URL can be checked using this method. For instance, you could allow only images being served from HTTPs using `{ protocol: "https" }`.
+
+See [`image.domains`](/en/reference/configuration-reference/#imagedomains) and [`image.remotePatterns`](/en/reference/configuration-reference/#imageremotepatterns) for detailed configuration instructions.
```
Comment (Yan-Thomas) [inline src/content/docs/en/guides/assets.mdx:235]:
```suggestion
See [`image.domains`](/en/reference/configuration-reference/#imagedomains-experimental) and [`image.remotePatterns`](/en/reference/configuration-reference/#imageremotepatterns-experimental) for detailed configuration instructions.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**